sensor/common/store/resolver/resolver.go

```diff
@@ -0,0 +1,12 @@
+package resolver
+
+import "github.com/stackrox/rox/sensor/common/store"
+
+type DeploymentReference func(store store.DeploymentStore) []string
+
+// ResolveDeploymentIds is an identify function that simply returns a list of deployment ids passed
+func ResolveDeploymentIds(ids ...string) DeploymentReference {
```

Comment:
I think we need to add the `action` somewhere. I'm not sure if here would be a good idea or maybe as a field in the `ResourceEvent`. The reason why is because the detector relies on the action to do different things and if I understood the code correctly we will send a `ResourceEvent` with nothing but the `DeploymentReference` to the detector, right?

Reply:
Addressed this in a follow-up PR. If you don't mind, I'd keep it there only: https://github.com/stackrox/stackrox/pull/3853